src/trivial_demonstrations/multiplication_table

```diff
@@ -0,0 +1,17 @@
+/* Prints out the multiplication table for a given number.
+ * Exemplifies the use of a for loop and a user defined function.
+ */
+
+
+fn main() {
+     use std::io::{stdin};//HEADER FILE TO TAKE INPUT
+     let mut num = String::new();
+    println!("Enter the number to print multiplication table for:");
+     stdin().read_line(&mut num).expect("Did not enter a correct string");//TAKING INPUTS AS ASTRING
+     let mut n = num.trim().parse::<i32>().unwrap();//CONVERTING STRING TO INTEGER
```

Comment:
Remove *mut* keyword from n. Do not use mut unless required. It prevent unwanted modification of variable values.